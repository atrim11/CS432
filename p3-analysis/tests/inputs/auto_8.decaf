def int multiply(int x, int y) {
    return x * y;
}

def bool main(int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a, int a) {
    // Parameter mismatch
    int eresult;
    int sum;
    result = multiply(5);  // Error: multiply expects 2 parameters

    result = factorial(5);  // Will cause infinite recursion
    result = invalidReturn();
}
