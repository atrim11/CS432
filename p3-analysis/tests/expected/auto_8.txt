Symbol 'result' undefined on line 9
Invalid number of function arguments on line 9
Symbol 'result' undefined on line 11
Symbol 'factorial' undefined on line 11
Symbol 'result' undefined on line 12
Symbol 'invalidReturn' undefined on line 12
Duplicate symbols named 'a' in scope started on line 5
'main' must return an integer
'main' must take no parameters
